sq/Makefile

```diff
@@ -36,6 +36,7 @@ install: build-release
 	$(MAKE) -C../store install
 
 test-dsm:
+	cargo run -- --version
```

Comment:
Only to build the test binary `target/debug/sq` and use that instead of `cargo run --` in every invocation (which may have warnings when debugging, etc).

Reply:
Ok, I see

Reply:
> do you think it would be good to, in the future, support certificate-based auth using a PKCS 8 private key and a X.509 certificate, instead of requiring a PKCS 12 bundle?

Yes, definitely, thanks for the link, I will follow it! I am not fancy of forcing the user to play around with private keys and introduce dependencies, even if it is only to create the PKCS12 identity (e.g. `openssl pkcs12 -export ...`).